pkg/arvo/sys/vane/gall.hoon

```diff
@@ -1874,6 +1879,18 @@
   ::
   ~|  [%gall-call-failed duct hic]
   =/  =task  ((harden task) hic)
+  =/  prov=path
```

Comment:
Ah ok, I hadn't seen this when I commented "do we need to upgrade wires?"  Now my question is: do we need to upgrade wires in `+take` in addition to here in `+call` -- maybe in `+mo-handle-use` or `+mo-handle-sys`?

Reply:
existing wire format is sufficient afaict. Maybe I'm misunderstanding what you're suggesting here?

Reply:
I thought the wire format had changed, but since it hasn't, there's no issue.